libvast/test/fixtures/events.cpp

```diff
@@ -92,6 +96,77 @@ events::copy(std::vector<table_slice_handle> xs) {
   return result;
 }
 
+class event_tracking_builder {
+  table_slice_builder_ptr inner_;
```

Comment:
[Use the order `public`, `proctected`, `private` for functions and members in classes.](https://github.com/vast-io/vast/blob/master/CONTRIBUTING.md#classes)